frontend/src/diagram/sprotty/views/LabelView.tsx

```diff
@@ -64,4 +68,22 @@ export class LabelView extends SLabelView {
     }
     return vnode;
   }
+  convertText(text: string, fontSize: number) {
```

Comment:
What is the return type of this function? Is that some JSX elements? If that's the case, then you have a function which takes some arguments or props and which returns some JSX... then it's a component :)

Reply:
I am not sure to understand what you expect.
I changed the code a little.
But perhaps you prefer something like this ?
```
{lines.length ===1? : text : lines.map()....}
```